src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,27 @@
 
 public class MainApp {
 
+	public static void main(String[] args) {
+
+		Bulldozer bulldozer = new Bulldozer();
+
+		Excavator excavator = new Excavator();
+
+		Truck truck = new Truck();
+
+
+		bulldozer.doWork();
+
+		excavator.doWork();
+
+		truck.doWork();
+
+
+		bulldozer.stopWork();
+
+		excavator.stopWork();
+
+		truck.stopWork();
```

Comment:
```suggestion

		Machine bulldozer = new Bulldozer();
		Machine excavator = new Excavator();
		Machine truck = new Truck();

		bulldozer.doWork();
		excavator.doWork();
		truck.doWork();

		bulldozer.stopWork();
		excavator.stopWork();
		truck.stopWork();
```

Тут слишком много пустых строк, тяжело читать код